### ISSUE TITLE
JSON and train.py

### ISSUE THREAD

**Issue by ExitPath** [SunnerLi/Cross-you-in-style#3]
Hi, excuse me, I am very interested in this work, yet I have two questions about it. 1. How to generate the JSON file, for example, train_music.json, portraittest_music.json, etc. 2. Can you provide the training code train.py? Thank you!

**Comment by SunnerLi** [comment]
Hi @ExitPath ,
thanks for your attention.
To generating JSON file, we don't provide code to generate automatically.
You can refer to explanation in readme to generate JSON file manually.
Also, we only need `sample_train_music_json` in order to perform inference.

At last, we are sorry that we don't have plan to release training code.
If you really eager want to reproduce the training result from scratch, 
you can write training code by your own :-)
we have described the network architecture, loss function and hyper-parameters in our paper.
